org.eclipse.xtext.builder.standalone/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -1,54 +1,65 @@
-#Wed Oct 14 22:36:24 CEST 2009
```

Comment:
Good to see you enable these, but shouldn't these be copy pasted to every xtext plugin?

Reply:
removed the file change from this change